### PR TITLE
CORL final version for flow 0.3.0

### PR DESCRIPTION
**Pull request by eugenevinitsky**
- A commit for the release of Flow 0.3.0 to freeze the code for CORL 2018

### CODE REVIEW

File: flow/benchmarks/baselines/merge012.py [hunk 1]
```diff
@@ -64,8 +64,8 @@ class needed to run simulations
                departLane="free", departSpeed=7.5)
 
     sumo_params = SumoParams(
-        restart_instance=False,
-        sim_step=0.2,  # time step decreased to prevent occasional crashes
+        restart_instance=True,
+        sim_step=0.5,  # time step decreased to prevent occasional crashes
```
Comment (AboudyKreidieh) [inline flow/benchmarks/baselines/merge012.py]: is this a mistake or something we want to do?

File: flow/config_default.py [hunk 2]
```diff
@@ -16,4 +16,4 @@
 
 AWS_ACCESS_SECRET = os.environ.get("AWS_ACCESS_SECRET", None)
 
-AWS_S3_PATH = "s3://bucket_name"
+AWS_S3_PATH = "s3://public.flow.results/corl_exps/exps_final/ppo/grid_0"
```
Comment (AboudyKreidieh) [inline flow/config_default.py]: change back to "bucket_name"

File: flow/visualize/visualizer_rllib.py [hunk 3]
```diff
@@ -110,6 +110,7 @@
     module = __import__("flow.envs", fromlist=[flow_params["env_name"]])
     env_class = getattr(module, flow_params["env_name"])
     env_params = flow_params['env']
+    env_params.evaluate = True
```
Comment (AboudyKreidieh) [inline flow/visualize/visualizer_rllib.py]: should we make this an input argument?

Comment (eugenevinitsky) [reply flow/visualize/visualizer_rllib.py]: Yeah, we should. It should probably default to false

Comment (eugenevinitsky) [reply flow/visualize/visualizer_rllib.py]: Otherwise, people will be confused why their trained reward is off

**Pull request merged**